docroot/profiles/custom/cgov_site/translations/cgov_site.es.po

```diff
@@ -14,6 +14,10 @@ msgctxt "PHP date format"
 msgid "F j, Y"
 msgstr "j \\d\\e F \\d\\e Y"
 
+msgctxt "PHP date format"
```

Comment:
This should be able to be reverted as well if `core.date_format.cgov_short_date is being reverted.